lib/parity/environment.rb

```diff
@@ -130,7 +130,9 @@ def git_remote
     end
 
     def heroku_app_name
-      git_remote.split(":").last.split(".").first
+      git_remote.
```

Comment:
I'm actually not sold on `%r{...}` because it adds more line noise than escaping the slash does. So I'm revising to vote for `.split(/[\/:]/)`